exercises/practice/rna-transcription/RnaTranscriptionTest.php

```diff
@@ -31,28 +9,93 @@ public static function setUpBeforeClass(): void
         require_once 'RnaTranscription.php';
     }
 
+    /**
+     * @testdox It transcribes guanine to cytosine
+     * uuid 123e4567-e89b-12d3-a456-426614174001
+     */
     public function testTranscribesGuanineToCytosine(): void
     {
         $this->assertSame('G', toRna('C'));
     }
 
+    /**
+     * @testdox It transcribes cytosine to guanine
+     * uuid 123e4567-e89b-12d3-a456-426614174002
+     */
     public function testTranscribesCytosineToGuanine(): void
     {
         $this->assertSame('C', toRna('G'));
     }
 
+    /**
+     * @testdox It transcribes thymine to adenine
+     * uuid 123e4567-e89b-12d3-a456-426614174003
+     */
     public function testTranscribesThymineToAdenine(): void
     {
         $this->assertSame('A', toRna('T'));
     }
 
+    /**
+     * @testdox It transcribes adenine to uracil
+     * uuid 123e4567-e89b-12d3-a456-426614174004
+     */
     public function testTranscribesAdenineToUracil(): void
     {
         $this->assertSame('U', toRna('A'));
     }
 
+    /**
+     * @testdox It transcribes all occurrences in a given sequence
+     * uuid 123e4567-e89b-12d3-a456-426614174005
+     */
     public function testTranscribesAllOccurrencesOne(): void
     {
         $this->assertSame('UGCACCAGAAUU', toRna('ACGTGGTCTTAA'));
     }
```

Comment:
`canonical-data.json` (https://github.com/exercism/problem-specifications/blob/main/exercises/rna-transcription/canonical-data.json) is **the source of test data** we use here in PHP track. That file has all data we need: the UUID, the description and the test cases.

All we do is write that data into our test file:

```suggestion
    /**
     * @testdox RNA complement
     * uuid 79ed2757-f018-4f47-a1d7-34a559392dbf
     */
    public function testTranscribesAllOccurrencesOne(): void
    {
        $this->assertSame('UGCACCAGAAUU', toRna('ACGTGGTCTTAA'));
    }
```

Do this for all tests. Remove the tests, that are not in `canonical-data.json`. Do not add new tests.